guides/release/components/helper-functions.md

```diff
@@ -208,9 +208,25 @@ For the full list of available helpers, you can check the [template helpers API
 ### The `get` helper
 
 The [`{{get}}`](https://api.emberjs.com/ember/release/classes/Ember.Templates.helpers/methods/get?anchor=get)
-helper makes it easy to dynamically send the value of a variable to another
-helper or component. This can be useful if you want to output one of several
-values based on the result of a getter.
+helper makes it easy to dynamically look up a property on an object or an element in an array. The second argument to `{{get}}` can be a string or an integer depending on the object being accessed.
```

Comment:
Although integer is technically correct, I think number may be easier to understand. I added a comma, to help break the sentence.

```suggestion
helper makes it easy to dynamically look up a property on an object or an element in an array. The second argument to `{{get}}` can be a string or a number, depending on the object being accessed.
```